resource_packs/files/utility/brewing_guide/ui/brewing_stand_screen.json

```diff
@@ -17,5 +17,11 @@
 				}
 			}
 		]
+	},
+	"brewing_stand_screen@common.inventory_screen_common": {
+		"variables": [],
```

Comment:
I did that so that in future if minecraft decides to add any new devices/new ui modes then it will still use the classic ui one, but i changed it according to your request @drav0011